tests/openedx_tagging/core/tagging/test_views.py

```diff
@@ -782,3 +743,21 @@ def test_tag_object_without_permission(self, user_attr, expected_status):
 
         response = self.client.put(url, {"tags": ["Tag 1"]}, format="json")
         assert response.status_code == expected_status
+        assert not status.is_success(expected_status)  # No success cases here
+
+    def test_tag_object_count_limit(self):
```

Comment:
Would super helpful if we could also add a test case that covers the scenario described above.

Reply:
Agreed! Thank you!

Done here: 432e4c22eb18a05eeafb5070a14666680f4c1484